Explorer/Assets/DCL/Utilities/WorldProxy.cs

```diff
@@ -0,0 +1,42 @@
+using Arch.Core;
+
+namespace DCL.Utilities
+{
+    public class WorldProxy
+    {
+        private World world;
+
+        public World GetWorld()
+        {
+            return world;
+        }
+
+        public void SetWorld(World newWorld)
+        {
+            world = newWorld;
+        }
+
+        public void Add(Entity entity, in object cmp)
```

Comment:
This method should be generic, otherwise whatever you pass here will be boxed. And then additional logic in `Arch` is called to infer the real type.
In all cases consider using generic APIs only

Reply:
Done!